OpenSSL missing for i686-musl

I haven't been able to compile it with `musl-gcc`.

```
$ bash /openssl.sh 1.0.2j linux-elf musl- -static
/usr/bin/ld: skipping incompatible ../libssl.a when searching for -lssl
/usr/bin/ld: skipping incompatible ../libssl.a when searching for -lssl
/usr/bin/ld: cannot find -lssl
/usr/bin/ld: skipping incompatible ../libcrypto.a when searching for -lcrypto
/usr/bin/ld: skipping incompatible ../libcrypto.a when searching for -lcrypto
/usr/bin/ld: cannot find -lcrypto
/usr/bin/ld: skipping incompatible /usr/local/lib/libc.a when searching for -lc
/usr/bin/ld: cannot find -lc
collect2: error: ld returned 1 exit status
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -13,7 +13,7 @@ matrix:
     - env: TARGET=armv7-unknown-linux-gnueabihf   CPP=1 DYLIB=1 STD=1 OPENSSL=0.5.5  RUN=1
     - env: TARGET=armv7-unknown-linux-musleabihf                STD=1                RUN=1
     - env: TARGET=i686-unknown-linux-gnu          CPP=1 DYLIB=1 STD=1 OPENSSL=0.5.5  RUN=1
-    - env: TARGET=i686-unknown-linux-musl                       STD=1                RUN=1
+    - env: TARGET=i686-unknown-linux-musl                       STD=1 OPENSSL=0.5.5  RUN=1
     - env: TARGET=mips-unknown-linux-gnu          CPP=1 DYLIB=1 STD=1 OPENSSL=0.5.5  RUN=1
     - env: TARGET=mips64-unknown-linux-gnuabi64   CPP=1 DYLIB=1 STD=1 OPENSSL=0.7.17 RUN=1
     - env: TARGET=mips64el-unknown-linux-gnuabi64 CPP=1 DYLIB=1 STD=1 OPENSSL=0.7.17 RUN=1

--- a/docker/i686-unknown-linux-musl/Dockerfile
+++ b/docker/i686-unknown-linux-musl/Dockerfile
@@ -1,4 +1,4 @@
-FROM ubuntu:16.04
+FROM i386/ubuntu:16.04
 
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
@@ -18,3 +18,12 @@ RUN apt-get install -y --no-install-recommends \
     bash /musl.sh 1.1.15 i386 -m32
 
 ENV CC_i686_unknown_linux_musl=musl-gcc
+
+COPY openssl.sh /
+RUN apt-get install -y --no-install-recommends \
+    g++-multilib && \
+    bash /openssl.sh linux-elf musl- -m32
+
+ENV OPENSSL_DIR=/openssl \
+    OPENSSL_INCLUDE_DIR=/openssl/include \
+    OPENSSL_LIB_DIR=/openssl/lib

